dspace/config/spring/api/orcid-authority-services.xml

```diff
@@ -40,9 +40,10 @@
         </property>
     </bean>
 
+
     <alias name="OrcidSource" alias="AuthoritySource"/>
-    <bean name="OrcidSource" class="org.dspace.authority.orcid.Orcid">
-        <constructor-arg value="http://pub.orcid.org"/>
+    <bean name="OrcidSource" class="org.dspace.authority.orcid.Orcidv2" init-method="init">
+        <constructor-arg value="${orcid.api.url}"/>
```

Comment:
This seems like it's the probably cause of your Travis unit test issues.  You have a placeholder `${orcid.api.url}` defined in this Spring Configuration, but that configuration is _not defined elsewhere in the DSpace configuration._

My guess would be that you should give this a default value in the `dspace.cfg` file like:

`orcid.api.url = http://pub.orcid.org`

Reply:
The failing of the unit tests was indeed because of the missing property.
I've currently added it to the dspace.cfg to atleast have a placeholder.

In the meantime I'd additionally also built with the license:check, which seemed fine locally, but the travis build is failing because of the embedded ORCID code, which DO contain the license header, but not of the correct format.

Reply:
Should we not exclude ORCID's code from the license check?

Reply:
The embedded ORCID code should be removed from our core API (ideally into a separate GitHub project, or similar).  I wouldn't worry about fixing the licenses, as the ORCID code must be extracted separately.  (We also should not be licensing code that we didn't write -- the ORCID code already has an MIT license.)